packages/terra-overlay/src/Overlay.jsx

```diff
@@ -126,27 +126,30 @@ class Overlay extends React.Component {
 
   enableContainerChildrenFocus() {
```

Comment:
Can we execute this logic in componentDidUpdate instead? There's probably a quirk with the inert implementation I'm missing, but from what I'm looking at here, it seems like the microtasks (TIL the technical term, thank you) would be unnecessary if we're doing the DOM manipulation in the expected lifecycle functions rather than render.

Reply:
Good call, updated in this commit: https://github.com/cerner/terra-core/pull/2679/commits/cd17e89e5a8b2b135b21c3569fa566ea147bcc6c